src/telepathy/telepathy-stubs.c

```diff
@@ -303,19 +303,6 @@ void sipe_backend_notify_error(SIPE_UNUSED_PARAMETER struct sipe_core_public *si
 			       SIPE_UNUSED_PARAMETER const gchar *title,
 			       SIPE_UNUSED_PARAMETER const gchar *msg) {}
 
-/** USER *********************************************************************/
-
-void sipe_backend_user_feedback_typing(SIPE_UNUSED_PARAMETER struct sipe_core_public *sipe_public,
```

Comment:
Making all in telepathy
Making all in data
libtool: link: gcc -Werror -Wall -Wextra -Waggregate-return -Wcast-align -Wdeclaration-after-statement -Wdeprecated-declarations -Winit-self -Wmaybe-uninitialized -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wpointer-arith -Wundef -Wunused-but-set-variable -DLOCALEDIR=\"/usr/local/share/locale\" -pthread -I/usr/include/telepathy-1.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -pthread -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I./../api -g -O2 -o telepathy-sipe telepathy_sipe-telepathy-buddy.o telepathy_sipe-telepathy-connection.o telepathy_sipe-telepathy-debug.o telepathy_sipe-telepathy-dnsquery.o telepathy_sipe-telepathy-main.o telepathy_sipe-telepathy-protocol.o telepathy_sipe-telepathy-schedule.o telepathy_sipe-telepathy-search.o telepathy_sipe-telepathy-status.o telepathy_sipe-telepathy-stubs.o telepathy_sipe-telepathy-tls.o telepathy_sipe-telepathy-transport.o  ../core/.libs/libsipe_core.a -lwinpr -lfreerdp-shadow -lfreerdp-shadow-subsystem -L/usr/lib64 ../purple/.libs/libsipe_backend_vv.a ../purple/.libs/libsipe_backend.a ../core/.libs/libsipe_core_crypto.a ../core/.libs/libsipe_core_libxml2.a ../core/.libs/libsipe_core_mime.a -lgmime-2.6 -lxml2 -lssl3 -lsmime3 -lnss3 -lnssutil3 -lplds4 -lplc4 -lnspr4 -lcrypto -ltelepathy-glib -ldbus-glib-1 -ldbus-1 -lgio-2.0 -lgobject-2.0 -lglib-2.0 -lpurple -lgssapi_krb5 -lkrb5 -lk5crypto -lcom_err -pthread
../purple/.libs/libsipe_backend.a(libsipe_backend_la-purple-user.o): In function `sipe_backend_user_feedback_typing':
/usr/local/src/sipe/src/purple/purple-user.c:48: multiple definition of `sipe_backend_user_feedback_typing'
telepathy_sipe-telepathy-stubs.o:/usr/local/src/sipe/src/telepathy/telepathy-stubs.c:309: first defined here
../purple/.libs/libsipe_backend.a(libsipe_backend_la-purple-user.o): In function `sipe_backend_user_feedback_typing_stop':
/usr/local/src/sipe/src/purple/purple-user.c:57: multiple definition of `sipe_backend_user_feedback_typing_stop'
telepathy_sipe-telepathy-stubs.o:/usr/local/src/sipe/src/telepathy/telepathy-stubs.c:311: first defined here
../purple/.libs/libsipe_backend.a(libsipe_backend_la-purple-user.o): In function `sipe_backend_user_ask':
/usr/local/src/sipe/src/purple/purple-user.c:70: multiple definition of `sipe_backend_user_ask'
telepathy_sipe-telepathy-stubs.o:/usr/local/src/sipe/src/telepathy/telepathy-stubs.c:316: first defined here
../purple/.libs/libsipe_backend.a(libsipe_backend_la-purple-user.o): In function `sipe_backend_user_close_ask':
/usr/local/src/sipe/src/purple/purple-user.c:87: multiple definition of `sipe_backend_user_close_ask'
telepathy_sipe-telepathy-stubs.o:/usr/local/src/sipe/src/telepathy/telepathy-stubs.c:317: first defined here
collect2: error: ld returned 1 exit status